data/preset/gallery.yaml

```diff
@@ -0,0 +1,555 @@
+                                                  other: CG5
+                  - type: ContentGroup.ContentGroup
+                    slots:
+                      arrangement:
+                        <<: *main-arrangement
+                      selection:
+                        <<: *selection-home
+                        id: CG7
+                        slots:
+                          filter:
+                            type: Filter.Articles
+                            slots:
+                              sub-filter:
+                                type: Filter.OtherGroup
+                                references:
+                                  other: CG1
+                                slots:
+                                  sub-filter:
+                                    type: Filter.OtherGroup
+                                    references:
+                                      other: CG2
+                                    slots:
+                                      sub-filter:
+                                        type: Filter.OtherGroup
+                                        references:
+                                          other: CG3
+                                        slots:
+                                          sub-filter:
+                                            type: Filter.OtherGroup
+                                            references:
+                                              other: CG4
+                                            slots:
+                                              sub-filter:
+                                                type: Filter.OtherGroup
+                                                references:
+                                                  other: CG5
+                                                slots:
+                                                  sub-filter:
+                                                    type: Filter.OtherGroup
+                                                    references:
+                                                      other: CG6
+
+- &set-page
+  type: Layout.TopMenu
+  slots:
+    top-menu:
+      <<: *main-menu
+    content:
+      shortdef: 'Layout.InfiniteScrolling(expand: true)'
+      references:
+        lazy-load: set-articles
+      slots:
+        content:
+          type: Layout.ResponsiveMargins
+          slots:
+            content:
+              shortdef: 'Layout.Box(orientation: vertical)'
+              slots:
+                contents:
+                  - shortdef: 'Banner.Set(halign: center)'
+                    slots:
+                      card:
+                        type: Card.Title
+                  - type: ContentGroup.ContentGroup
+                    slots:
+                      arrangement:
+                        <<: *main-arrangement
+                      selection:
+                        <<: *selection-set
+                        id: CG1
```

Comment:
continuing with my previous comment, "id: CG1" is defined twice, but should probably be unique.